### ISSUE TITLE
Discussion on ransomware_groups.md

### ISSUE THREAD

**Issue by idarlund** [fastfire/deepdarkCTI#740]
When cleaning up ransomware markdown file (https://github.com/fastfire/deepdarkCTI/pull/739) I discovered that someone added a threat intel company to it.

In that pull request I comment; "we should also have a discussion on what we want in this file; in my opinion this file should contain URLs to the ransomware threat actors and maybe other publicly available lists on these threat actors. what it should not contain in my opinion is "disguised marketing" for threat intel companies such as eCrime Services and others."

I propose that we do that in this issue.

**Comment by fastfire** [comment]
@idarlund I agree with what you said. There are 2 options: either we put the services/platforms in the file https://github.com/fastfire/deepdarkCTI/blob/main/others.md, or we create a new file calling it products/services or something like that. I propose to discuss this in the Telegram channel.

**Comment by idarlund** [comment]
removing eCrime "advertisment" https://github.com/fastfire/deepdarkCTI/pull/747

**Comment by idarlund** [comment]
eCrime started following me on twitter after this PR 🤣
![signal-2024-11-17-100849](https://github.com/user-attachments/assets/fba4ba31-1b9b-4d4c-8e5e-38e5fc8a6dd1)

anyway.. there's been a discussion on telegram, and hence; removing advertisements like this from the markdown file
![Screenshot from 2024-11-17 10-11-12](https://github.com/user-attachments/assets/5852d83a-1d12-4cc1-800f-5fb8e25a4107)
![Screenshot from 2024-11-17 10-11-30](https://github.com/user-attachments/assets/03a2727b-d84b-49ff-9776-c85b9699832d)
![Screenshot from 2024-11-17 10-11-52](https://github.com/user-attachments/assets/0710fecb-b12f-4880-909f-20a914bc804a)
